edb/edgeql/ast.py

```diff
@@ -281,6 +281,10 @@ class BinOp(Expr):
     right: Expr
 
 
+class SetUnionOp(BinOp):
```

Comment:
I'd name this `SetConstructorOp` for clarity (and add a comment that it's needed for `{...}`)

Reply:
Good idea, updated.